README.md

```diff
@@ -16,14 +16,30 @@ to your `devcontainer.json`:
 }
 ```
 
-Then launch your Codespace. After it starts up, run [`tailscale up`](https://tailscale.com/kb/1080/cli/#up):
+## Starting Tailscale
+
+The Tailscale daemon starts automatically as part of the devcontainer entrypoint.
+
+### Manual Login
+
+You will need to login to tailscale, which can either be done manually like so:
```

Comment:
nit: "log in"

There is an "either" but I do not see an "or". Was the link below meant to be that?